packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -37,7 +37,20 @@ describe('request-queue tests', () => {
     chai.assert.isObject(queue._config);
   });
 
-  it('queueName is corrent', () =>{
+  it('initialize should re-fill the queue', async () => {
```

Comment:
oh I can cover that. However earlier on reboot of sw the queue was not getting re-initialized, which this should cover now.